Makefile

```diff
@@ -137,7 +137,6 @@ test-misc-integration: test-bins
 	'Test_RestoreFromPassedCSV|'\
 	'Test_Recover_PassedSwap_BTC|'\
 	'Test_Recover_PassedSwap_LBTC|'\
-	'Test_Wumbo|'\
```

Comment:
Is the wumbo test also used in a different place or can we remove the test completely from the `tests`?

Reply:
I didn't feel 100% comfortable removing `wumbo_test.go` too just in case it was used elsewhere. Pretty certain it is fine to remove it though.